docs/en/setup/backend/backend-cluster.md

```diff
@@ -85,7 +85,7 @@ the yml to active.
 cluster:
   consul:
     serviceName: ${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
-    # Consul cluster nodes, example: 10.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
+    # The consul cluster agents(agent is server agent or client agent). example: 127.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
```

Comment:
If you are going to config the agent `ip:port`, then is it still necessary to have multiple addresses?  Or those addresses should be server agent addresses?

Reply:
`127.0.0.1:8500` and `10.0.0.2:8500` seems strange together. Should change this to 
```
example, 1. client agent, 127.0.0.1:8500, 2. server agent, 127.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
```

Reply:
ok. I will modify it